joomla/joomla-cms
# [NEW FEATURE] Adding Edit Associations Toolbar

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/21003

### Summary of Changes
Adding a new toolbar for items which can be associated.
The toolbar button redirects to the side by side Multilingual associations component.
Changes include new methods in FormController as in AdminModel.
And the new toolbar button for each view concerned.

### Testing Instructions
Install a multilngual site. Make sure Associations are set.
Edit any

- article
- contact
- menu item
- newsfeed
- category

Click on the new `Edit Associations` Toolbar button. Example:
<img width="904" alt="screen shot 2018-07-09 at 11 57 31" src="https://user-images.githubusercontent.com/869724/42444311-02a9e928-8370-11e8-8e44-319f17ecf69e.png">

You should be redirected to the side by side.
<img width="1040" alt="screen shot 2018-07-09 at 12 08 12" src="https://user-images.githubusercontent.com/869724/42444605-cb213f96-8370-11e8-98eb-f4a8d9c3c5c1.png">

If the item is set to ALL languages, a Notice will display.
<img width="1176" alt="screen shot 2018-07-09 at 12 09 16" src="https://user-images.githubusercontent.com/869724/42444663-f3c5bac6-8370-11e8-9691-f1eb5c0dd374.png">


### Documentation Changes Required
I guess so. Target is 3.9.x

@bakual @rdeutz @alikon
